src/odemis/driver/scanner.py

```diff
@@ -201,30 +209,36 @@ def updateMetadata(self, md):
     def getMetadata(self):
         return self._external_scanner.getMetadata()
 
-    def claimBeam(self, claim):
+    def claimBeam(self, claim: bool, user: object):
```

Comment:
I really appreciate this addition of parameter type, makes it much better readable and forces anyone to use the proper type upfront.